.NET/Microsoft.Recognizers.Text.DateTime/Utilities/TasksModeSetHandler.cs

```diff
@@ -0,0 +1,292 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using Microsoft.Recognizers.Text.DataTypes.TimexExpression;
+using DateObject = System.DateTime;
+
+namespace Microsoft.Recognizers.Text.DateTime.Utilities
+{
+    public static class TasksModeSetHandler
+    {
+        public static DateTimeResolutionResult TasksModeResolveSet(ref DateTimeResolutionResult result, string innerTimex, DateTimeParseResult pr = null)
+        {
+            result.Timex = innerTimex;
+
+            result.FutureValue = result.PastValue = "Set: " + innerTimex;
```

Comment:
"Set: " + innerTimex
should be the output of a well-named method instead of direct string manipulation here.
This will make it easirt for future changes that modify the value generation in a single place.

Reply:
I followed the convention of the SetHandler.cs ResolveSet method, and this step is only used once. The following method is extension for ResolveSet for TasksMode.

Reply:
Add at least a `//@TODO` comment here that this should be changed.

Reply:
Replace the follwing statement ("Set: " + innerTimex;) with the method call and constant name for set.